charts/selenium-grid/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
 version: 0.25.1
-appVersion: 4.15.0-20231110
+appVersion: 4.15.0-20231122
```

Comment:
I updated this appVersion to be aligned with the whole project. In the next `[deploy]`, text replacement will occur properly

Reply:
Makes sense. 👍